include/backend/operations.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <stdexcept>
+
+#include "backend/type.h"
+
+namespace celerity::detail::backend_detail {
```

Comment:
I'm not sure what is happening here, for some reason `clang-tidy` (and my local `clangd`) for that matter is confused by the backend system, maybe because it uses a nested CMakeLists...?

Reply:
nvm, I forgot to set the `CXX_STANDARD` on `celerity_backends`.